src/platforms/common/enriching-events/screenshots.mdx

```diff
@@ -15,6 +20,9 @@ The Screenshots feature is available in Beta. Beta features are still in-progres
 When a user experiences an error, Sentry provides the ability to take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink>.
 
 This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. So inherently, this feature will be a best effort solution.
+<PlatformSection supported={["apple.io"]}>
+In iOS is not possible to take a screenshot during hard crashes, only manually captured errors or exceptions will contain a screenshot.
+</PlatformSection>
```

Comment:
```suggestion

<PlatformSection supported={["apple.ios"]}>

On iOS, it's not possible to take a screenshot during hard crashes, only manually captured errors or exceptions will contain a screenshot.

</PlatformSection>
```

You can use the preview to check if the content is correct or not: https://sentry-docs-git-feat-ios-screenshot.sentry.dev/platforms/apple/guides/ios/enriching-events/screenshots/